### PR TITLE
Ketch

### PR DESCRIPTION
**Pull request by vfarcic**
Adding Ketch

### CODE REVIEW

**Comment by AloisReitbauer** [comment]
This example is not using the podtato demo app and runs the individual use cases you can find the steps [here](https://github.com/cncf/podtato-head/blob/main/contributing.md)

File: delivery/ketch/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+```bash
+# Add a pool with ingress Traefik (default), replace ingress IP address by your ingress IP address
+ketch pool add mypool  --ingress-service-endpoint 35.247.8.23 --ingress-type traefik
+
+# Create app
+ketch app create bulletinboard --pool mypool       
+
+# Deploy app
+ketch app deploy bulletinboard -i docker.io/shipasoftware/bulletinboard:1.0
```
Comment (AloisReitbauer) [inline delivery/ketch/README.md:91]: This deploys the wrong application

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]

**Comment by vfarcic** [comment]
My bad... Let me do it again...

**Comment by AloisReitbauer** [comment]
Great! Any ETA? I would need to create a tag before moving towards the multi-service version of podtato-head.

**Comment by vfarcic** [comment]
Sorry for the delay. I'll work on it today and update the PR ASAP.

**Comment by vfarcic** [comment]
There is a problem that prevents me from using the app and the Dockerfile as-is with Ketch.

Ketch assumes that there is `entrypoint` in Dockerfile or, if there isn't, it "injects" `command: /bin/sh ...` into manifests. Since the final image is based on `scratch`, there is no `sh`.

Do you think it would be possible to modify the Dockerfile and change:

```
CMD ["/podtatoserver", "--version", "0.1.0" ]
```

to 

```
ENTRYPOINT ["/podtatoserver"]

CMD ["--version", "0.1.0" ]
```

That probably shouldn't affect any of the existing instructions.

**Comment by vfarcic** [comment]
We fixed the issue in Ketch. Now it should work. I'll update the PR before the end of the week.

**Comment by vfarcic** [comment]
@AloisReitbauer I just updated the PR to use podtato. Can you please take a look and let me know if it's okay or if there is something missing.

**Comment by AloisReitbauer** [comment]
Hello, trying the example on Kind. Many people have requested they run on kind.

One thing I noticed: 

http://podtato.127.0.0.1.nip.io.shipa.cloud is shown as the URL for the app. I added 127.0.0.1.nip.io as my local domain. I don't think the shipa.cloud part should be there

**Comment by vfarcic** [comment]
@AloisReitbauer I just updated it to use k3d, mostly because I already had an example with it. Is that OK or you have a strong preference towards kind?

**Comment by vfarcic** [comment]
How does it look? Is there anything else I can do to improve it?

**Comment by vfarcic** [comment]
@AloisReitbauer Please let me know if there is anything missing or wrong.

**Comment by AloisReitbauer** [comment]
Sorry this took so long. Approving and re checking the workflow

**Pull request merged**